google/cloud/bigtable/instance_admin_client.cc

```diff
@@ -322,13 +325,20 @@ class DefaultInstanceAdminClient : public InstanceAdminClient {
         stub->AsyncGetOperation(context, request, cq).release());
   }
 
+ protected:
+  std::shared_ptr<bigtable_admin::BigtableInstanceAdminConnection> connection()
```

Comment:
This changes the access modifier for `connection()`, that is Okay, but is it intentional?  Does the derived class need to use it?  If not, (seems that way) keep it `private`?

Reply:
Argh. This was an oversight. Thanks for catching it.